### PR TITLE
nunique()

### PR DESCRIPTION
**Pull request by crusaderky**

### CODE REVIEW

File: dask_expr/_collection.py [hunk 1]
```diff
@@ -1164,6 +1182,16 @@ def product(self):
     def unique(self):
         return new_collection(Unique(self.expr))
 
+    def nunique(self, dropna=True):
+        uniqs = self.drop_duplicates()
+        if dropna:
+            # count mimics pandas behavior and excludes NA values
+            if isinstance(uniqs, Index):
+                uniqs = uniqs.to_series()
```
Comment (crusaderky) [inline dask_expr/_collection.py:1190]: - XREF https://github.com/dask-contrib/dask-expr/issues/475

File: dask_expr/_collection.py [hunk 2]
```diff
@@ -1109,6 +1109,20 @@ def iloc(self):
...
+        if axis == 1:
+            return new_collection(
+                expr.NUniquePerBlock(self.expr, axis=axis, dropna=dropna)
+            )
+        else:
+            return concat(
+                [
+                    col.nunique(dropna=dropna).to_series(col.name)
+                    # for col in self  # TODO not implemented
+                    for col in (self.iloc[:, i] for i in range(self.shape[1]))
```
Comment (phofl) [inline dask_expr/_collection.py:1122]: Can you write this as

```
self.iloc[:, I].(dropna=dropna).to_series(col.name) for i in range(len(self.columns))
```
?

Comment (crusaderky) [reply dask_expr/_collection.py:1122]: Could you merge https://github.com/dask-contrib/dask-expr/pull/480?

Comment (crusaderky) [reply dask_expr/_collection.py]: done

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx

**Pull request merged**